src/stan/lang/ast/expr_type.hpp

```diff
@@ -2,6 +2,13 @@
 #define STAN_LANG_AST_EXPR_TYPE_HPP
 
 #include <stan/lang/ast/base_expr_type.hpp>
+#include <stan/lang/ast/ill_formed_type.hpp>
+#include <stan/lang/ast/void_type.hpp>
+#include <stan/lang/ast/int_type.hpp>
+#include <stan/lang/ast/double_type.hpp>
+#include <stan/lang/ast/vector_type.hpp>
+#include <stan/lang/ast/row_vector_type.hpp>
+#include <stan/lang/ast/matrix_type.hpp>
```

Comment:
I find these easier to deal with in alphabetical order.

[optional change]

Reply:
fixed.